code/game/atoms_movable.dm

```diff
@@ -98,6 +95,9 @@
 
 	break_all_tethers()
 
+	for(var/atom/movable/AM in src)
```

Comment:
This doesn't matter too much but this was my first step at fixing the burnables misbehaving problem, by just deleting this *before* moving them to nullspace. It works (kinda) but there are cases where this isn't enough (like stacks).